README.md

```diff
@@ -1 +1,32 @@
-##Hello
+# LAP 1 Code Challenge - GOOGLE
+
+## Installation & usage
+
+### Installation
+ - Clone or download the repo.
+ - Open terminal and navigate to server folder.
+ - Run npm install to install dependencies.
+ - (Optional) Get a google search api key and the cx and substitude the one that are commented in Client/script_index.js and in Client/script_result.js  for yours ones.
+ ### Usage 
+ - Open terminal and navigate to server folder.
+ - Run npm start to launch server
+ - Navigate to client folder.
+ - Start a server in the client usign for example python -m http.server
+
+ ## Technologies 
+ - JavaScript
+ - HTML, CSS
+ - node
+ - express
+ - cors 
+
+ ## Process
+ - you can write whatever you want to search
+ - the Button must only work after you have written something
+ - if its between this words: Spain, Music, Football it will send you to our own data base of links.
+ - if its not between those words and you have a google key it will get elements from your search in google
+ - its not between those words and you don't have a google key it will give you a message that we can't retrieve data.
+ 
```

Comment:
Very nice and clear readme, this really made it clear what to do and how to run the repo.